### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by notdeadyett** [qdm12/gluetun#1840]
### Is this urgent?

No

### Host OS

Unraid

### CPU arch

x86_64

### VPN service provider

ProtonVPN

### What are you using to run the container

docker run

### What is the version of Gluetun

Latest

### What's the problem 🤔

Constantly getting a warning in the logs about the internal port differing from the external port when using protonvpn port forwarding with wireguard.

### Share your logs

```plain text
2023-09-02T00:12:11Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
2023-09-02T00:12:11Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
2023-09-02T00:12:56Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
2023-09-02T00:12:56Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
2023-09-02T00:13:41Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
2023-09-02T00:13:41Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
2023-09-02T00:14:26Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
2023-09-02T00:14:26Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
2023-09-02T00:15:11Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
2023-09-02T00:15:11Z WARN [port forwarding] internal port assigned 0 differs from e
xternal port assigned 50326
```

### Share your configuration

```yml
docker run -d -p 8112:8112 --cap-add=NET_ADMIN --restart=always --name=gluetun -e VPN_SERVICE_PROVIDER=custom -e VPN_TYPE=wireguard -e VPN_ENDPOINT_IP=<redacted> -e VPN_ENDPOINT_PORT=51820 -e WIREGUARD_PUBLIC_KEY=<redacted> -e WIREGUARD_PRIVATE_KEY=<redacted> -e WIREGUARD_ADDRESSES="<redacted>" -e VPN_PORT_FORWARDING=on -e VPN_PORT_FORWARDING_PROVIDER=protonvpn qmcgaw/gluetun
```

**Comment by privacyguy123** [comment]
Noticed this bug too - only difference is I have the wg0.conf file passed through as a variable. Not sure what this error message means?

**Comment by miiraheart** [comment]
I am having the same warning as well

**Comment by boydwold** [comment]
Same for me.

**Comment by jgramling17** [comment]
Same here

**Comment by maxou2600** [comment]
Same here

**Comment by Landrin201** [comment]
I'm seeing the same; just noticed it today. No idea what's causing it. Also using ProtonVPN.

**Comment by qdm12** [comment]
TLDR Something to ignore, and check got changed in c0062fb8078e46466f03437233c8ab503e40231b from checking assignedInternalPort != assignedExternalPort to assignedInternalPort != requestedInternalPort

For context, gluetun requests internal port `0` and external port `0`, and the protonvpn natpmp server responds with an assigned internal port `0` and assigned external port `9876` (where 9876 is any port available). During testing, it didn't seem like the assigned internal port was differing from the assigned external port, but maybe this changed since then. Anyway, it does not matter, now the assigned internal port is left to `0`, and what really matters is the assigned external port anyway.

Also @notdeadyett please set a title to the issue next time, this really hardens filtering issues for me.

**Comment by consenti** [comment]
I updated to latest image, but still see the same error. 
Running version latest built on 2023-09-13T14:23:59.922Z (commit c0062fb)

**Comment by notdeadyett** [comment]
> I updated to latest image, but still see the same error.
> 
> Running version latest built on 2023-09-13T14:23:59.922Z (commit [c0062fb](https://github.com/qdm12/gluetun/commit/c0062fb8078e46466f03437233c8ab503e40231b))
> 
> Actually, I see two errors now 1 - internal port assigned 41662 differs from internal port requested 0 2 - internal port assigned 0 differs from external port assigned 41662

I am seeing the same.

**Comment by boydwold** [comment]
I get the same warning over and over.

**Comment by qdm12** [comment]
Re-fixed in ea25a0ff89b716c4ab7dd79c631b554828c65f9d

Thanks @fizzxed for finding the second log occurence (and also re-creating another issue since I don't monitor closed issues- shame on me!). From now on, it will only log a warning when the assigned external port changes from its previous value.
